composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "symfony-bundle",
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": "~2.3",
```

Comment:
Looking at the changes I seems that they would work with Symfony 2.7, 2.8 too, so the version should be 
`"symfony/symfony": "~2.7|~3.0"`